src/v2/Apps/Categories/Components/StickyNav.tsx

```diff
@@ -0,0 +1,102 @@
+import {
+  Link,
+  Pill,
+  Spacer,
+  themeProps,
+  Swiper,
+  SwiperCell,
+  SwiperRail,
+  SwiperCellProps,
+  SwiperRailProps,
+} from "@artsy/palette"
+import React from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+import { useNavBarHeight } from "v2/Components/NavBar/useNavBarHeight"
+import { extractNodes } from "v2/Utils/extractNodes"
+import { useMatchMedia } from "v2/Utils/Hooks/useMatchMedia"
+import { useWindowSize } from "v2/Utils/Hooks/useWindowSize"
+import { scrollIntoView } from "v2/Utils/scrollHelpers"
+import { StickyNav_geneFamiliesConnection } from "v2/__generated__/StickyNav_geneFamiliesConnection.graphql"
+interface StickyNavProps {
+  geneFamiliesConnection: StickyNav_geneFamiliesConnection
+}
+
+const StickyNav: React.FC<StickyNavProps> = props => {
+  const { geneFamiliesConnection } = props
+  const geneFamilies = extractNodes(geneFamiliesConnection)
+  const { mobile, desktop } = useNavBarHeight()
+  const isMobile = useMatchMedia(themeProps.mediaQueries.xs)
+  const navBarHeight = isMobile ? mobile : desktop
+  const stickyNavHeight = 50
+  const offset = navBarHeight + stickyNavHeight + 20
+
+  const handleClick = e => {
+    e.preventDefault()
+    const id = e.currentTarget.hash
+
+    scrollIntoView({ selector: id, offset, behavior: "smooth" })
+  }
+
+  return (
+    <>
+      <Spacer pb={1} />
+      <Swiper Cell={Cell} Rail={Rail}>
+        <Spacer pr={[2, 4]} />
+        {geneFamilies.map((geneFamily, i) => {
+          return (
+            <>
+              <Link
+                href={`#jump--${geneFamily.slug}`}
+                noUnderline={true}
+                onClick={handleClick}
+                key={geneFamily.slug}
+              >
+                <Pill>{geneFamily.name}</Pill>
+              </Link>
+              {i !== geneFamilies.length - 1 ? <Spacer pr={1} /> : null}
+            </>
+          )
+        })}
+        <Spacer pr={[2, 4]} />
+      </Swiper>
+      <Spacer pb={1} />
+    </>
+  )
+}
+
+const Cell: React.ForwardRefExoticComponent<SwiperCellProps> = React.forwardRef(
+  (props, ref) => {
+    return (
+      <SwiperCell
+        {...props}
+        ref={ref as any}
+        display="inline-flex"
+        verticalAlign="top"
+        pr={0}
+      />
+    )
+  }
+)
+
+const Rail: React.FC<SwiperRailProps> = props => {
+  const { width: windowWidth } = useWindowSize()
+  // This almost works, but not on the first render :(
+  const left = windowWidth > 1920 ? `${(windowWidth - 1920) / 2}px` : "auto"
+  return (
+    <SwiperRail {...props} display="block" position="relative" left={left} />
+  )
+}
```

Comment:
@dzucconi - you might be able to advise here

Reply:
Wait what's this doing? We should just skip all the sticky stuff here; which should simplify things. I can take a closer look on Monday though.

Reply:
Just pushed up some updated code - from the designs, this rail should be stuck to the top on scroll

Reply:
So yeah; I wouldn't use a Swiper here at all. Just use `HorizontalOverflow`

Reply:
Cool, that definitely reduces code and complexity. Q for you @dzucconi - the scrollbar appears on HorizontalOverflow, and it seems like it shouldn't based on [this line](https://github.com/artsy/palette/blob/c5b11add7b8b28ae210473956532c84d318bd947/packages/palette/src/elements/HorizontalOverflow/HorizontalOverflow.tsx#L49). When I inspect the bar, I actually don't see any `scrollbar-width` property set in the CSS (I would expect to see it on `Viewport`):

<img width="1374" alt="Screen Shot 2021-09-23 at 2 23 44 PM" src="https://user-images.githubusercontent.com/5361806/134506083-92d7a1ac-1b21-4b3c-a377-4de0a3535561.png">

Any idea what might be up here? IMO this shouldn't block this PR, but would be nice to sort out.